### PR TITLE
boards: I2C Feature for nucleo-f401

### PR DESCRIPTION
**Pull request by phectori**
Added I2C pin configuration for the board "Nucleo-F401"

### CODE REVIEW

**Comment by phectori** [comment]
ping @katezilla

**Comment by katezilla** [comment]
Congrats on your first PR :-) 
looks nice to me, but i don't have the board to test it.

**Comment by phectori** [comment]
Thanks, the i2c works for me in combination with an mpu9150. Should I show this to you in some way?

**Comment by OlegHahm** [comment]
@PeterKietzmann, @thomaseichinger, @haukepetersen, I guess one of you have the board to test?
@phectori, could you rebase, please

**Comment by phectori** [comment]
Rebased

**Comment by OlegHahm** [comment]
Thanks!

**Comment by PeterKietzmann** [comment]
Nope, we don't have a nucleo-f401. It's just a configuration file to an existing driver. So it should be kind of easy to test for someone with a board. Any volunteers :-) ?

File: boards/nucleo-f401/Makefile.features [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Put defined MCU peripherals here (in alphabetical order)
 FEATURES_PROVIDED += periph_cpuid
+FEATURES_PROVIDED += cpp
+FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_i2c
```
Comment (haukepetersen) [inline boards/nucleo-f401/Makefile.features]: cpp and uart are already present below...

**Comment by haukepetersen** [comment]
I also don't have the board, but the configuration looks sane.

File: boards/nucleo-f401/Makefile.features (hunk 1 above)
Comment (phectori) [inline boards/nucleo-f401/Makefile.features]: Excuse me, must have been the rebase gone wrong. Fixed now

**Comment by katezilla** [comment]
@phectori As Nucleo fxxx's do have the same Layout and it looks like the implementation of the f303 i'd belive you that it works. ACK from me. @PeterKietzmann  @OlegHahm  @haukepetersen,  Can i merge it if no one volunteers to test it or do i need another ACK?

**Comment by haukepetersen** [comment]
merge at will once Travis is happy

**Comment by gebart** [comment]
Travis is happy -> go

**Comment by OlegHahm** [comment]
@phectori, congrats to your first contribution to RIOT (if I'm not mistaken).

**Comment by phectori** [comment]
@OlegHahm, Thank you, hope there are many more to come. I want to try to write a driver for the mrf24j40ma module. Or is it already in development?

**Comment by OlegHahm** [comment]
There were several people asking for it on the mailing list and I think some people started to work on this. Please check:
https://lists.riot-os.org/pipermail/devel/2014-June/thread.html#808
https://lists.riot-os.org/pipermail/devel/2016-February/003463.html

**Comment by PeterKietzmann** [comment]
@phectori currently there are two guys working on the driver: @TobiasFredersdorf and "Bernhard Nägele". If you have further questions you should state them on the devel mailing list or get in contact with them directly.

**Pull request merged**